koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -185,9 +185,17 @@ def create_partitioned_ocp_on_cloud_parquet(self, data_frame, parquet_base_filen
 
         for usage_day in unique_usage_days:
             usage_date = pd.to_datetime(usage_day).date()
+            # Uniqueify manifest report counter dict entry with p for parquet (dont colide with csvs)
+            # We use the manifest to hold dated file counts for multiple workers processing the same dates in files
+            base_filename = f"{usage_date}_p"
+            counter = ReportManifestDBAccessor().update_and_get_parquet_batch_counter(base_filename, manifest_id)
+            # Parquet base filename dates here DO NOT match the data written to them
+            base_file_date = parquet_base_filename.split("_")[0]
+            non_date_base_name = parquet_base_filename.removeprefix(base_file_date)
+            usage_day_file_name = f"{usage_date}{non_date_base_name}"
```

Comment:
One side affect we may need to double check is OCP on GCP! We mentioned recently that the Normal flow for GCP (non ingress) does NOT overwrite parquet files because the data is unique based on what we gather from big query. This may in fact overwrite GCP files now for this flow since the file names will not be (current day date)+batch_num.

That being said the ingress flow WAS likely expecting files to be overwritten here. And working through this I'm fairly sure thats the duplication issue I've been looking at for you know who :car: 

If both of the above is the case we may want to just add a GCP ingress check here for NOW to handle both methods. I will say I do think all of this needs a rethink in the grand scheme of things eventually though.

Reply:
I have confirmed this is why :car: data was being duplicated in gcp_openshift_daily!

Reply:
I also confirmed that the non ingress flow _shouldnt_ overwrite files because of our naming convention here: `day_file = f"{invoice_month}_{partition_date}_{file_name}"` We append the daily file name to the end of the file, and this is used in all our parquet base naming.